src/main/java/at/hannibal2/skyhanni/config/features/GUIConfig.java

```diff
@@ -56,6 +56,26 @@ public static class TextBoxConfig {
     @Expose
     public Position realTimePosition = new Position(10, 10, false, true);
 
+    @Expose
+    @ConfigOption(name = "In-game Date", desc = "Show the in-game date of SkyBlock (like in Apec, §ebut with mild delays§7).\n(Though this one includes the SkyBlock year!)")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean inGameDateDisplay = true;
+
+    @Expose
+    public Position inGameDateDisplayPosition = new Position(10, 10, false, true);
+
+    @Expose
+    @ConfigOption(name = "In-game Date Refresh Rate", desc = "Change the amount of time in seconds you would like to refresh the in-game time display.")
+    @ConfigEditorSlider(
+            minValue = 1,
+            maxValue = 60,
+            minStep = 1
+    )
+    @ConfigAccordionId(id = 2)
+    public int inGameDateDisplayRefreshSeconds = 10;
+
```

Comment:
Please make this into a section in the config (see above this in the file) and also remove the configAccoridianId as its not needed here